### PR TITLE
[FIX] Bug on starting Jitsi video calls , multiple messages

### PR DESCRIPTION
**Pull request by ggazzo**

### CODE REVIEW

File: app/videobridge/client/views/videoFlexTab.js [hunk 1]
```diff
@@ -59,7 +60,7 @@ Template.videoFlexTab.onRendered(function() {
 		const update = () => {
 			const { jitsiTimeout } = Rooms.findOne({ _id: rid }, { fields: { jitsiTimeout: 1 }, reactive: false });
 
-			if (jitsiTimeout && (new Date() - new Date(jitsiTimeout) + CONSTANTS.TIMEOUT < CONSTANTS.DEBOUNCE)) {
+			if (jitsiTimeout && (TimeSync.TimeSync.serverTime(), - new Date(jitsiTimeout) + CONSTANTS.TIMEOUT < CONSTANTS.DEBOUNCE)) {
```
Comment (sampaiodiego) [inline app/videobridge/client/views/videoFlexTab.js]:
```suggestion
			if (jitsiTimeout && (TimeSync.serverTime(), - new Date(jitsiTimeout) + CONSTANTS.TIMEOUT < CONSTANTS.DEBOUNCE)) {
```

File: app/videobridge/client/views/videoFlexTab.js [hunk 2]
```diff
@@ -59,7 +60,7 @@ Template.videoFlexTab.onRendered(function() {
 		const update = () => {
 			const { jitsiTimeout } = Rooms.findOne({ _id: rid }, { fields: { jitsiTimeout: 1 }, reactive: false });
 
-			if (jitsiTimeout && (new Date() - new Date(jitsiTimeout) + CONSTANTS.TIMEOUT < CONSTANTS.DEBOUNCE)) {
+			if (jitsiTimeout && (TimeSync.serverTime(), - new Date(jitsiTimeout) + CONSTANTS.TIMEOUT < CONSTANTS.DEBOUNCE)) {
```
Comment (Bjeaurn) [inline app/videobridge/client/views/videoFlexTab.js:63]: So if there's a bug somewhere in the interface that keeps triggering new Jitsi Video calls, we've now not fixed that issue but slowed it down?

I'm automatically creating a new Jitsi video call every morning apparently and I have no idea why. Unsure about the RocketChat version we're running though, will have to ask about that.

**Comment by MetahumanFamily** [comment]
Issue still persisting. Rocket.Chat v 3.0.10. Jitsi is latest build as of today.

**Comment by luixxiul** [comment]
> Issue still persisting.

Do you mean this issue exactly? https://github.com/RocketChat/Rocket.Chat/issues/12148

**Comment by MetahumanFamily** [comment]
> > Issue still persisting.
> 
> Do you mean this issue exactly? #12148

Correct.

**Comment by luixxiul** [comment]
I think you would like to follow the issue over there as there seems to be another case: https://github.com/RocketChat/Rocket.Chat/issues/12148#issuecomment-620411251

**Comment by grimuz** [comment]
Same problem on:
Server: Rocket Chat 3.3.0
Client: Desktop App 2.17.9 (Windows)

**Comment by cremfert** [comment]
Same here, tries to start video conference repeatingly when MacBook is in standby.

**Pull request merged**